ModList/UI/Components/MarkdownText.cs

```diff
@@ -101,26 +101,31 @@ public UnityRenderer Renderer
 
         private (TMP_FontAsset font, string padding) LoadConfigFont(ModListConfig config)
         {
-            TMP_FontAsset asset = null;
+            TMP_FontAsset? asset = null;
             if (config.MonospaceFontPath != null)
             {
                 if (!File.Exists(config.MonospaceFontPath))
-                    Logger.md.Warn($"File '{config.MonospaceFontPath}' does not exist");
+                {
+                    // TODO: Inject logger for this
+                    // Logger.md.Warn($"File '{config.MonospaceFontPath}' does not exist");
+                }
                 else
                 {
-                    var ufont = new Font(config.MonospaceFontPath);
-                    asset = Helpers.TMPFontFromUnityFont(ufont);
+                    var uFont = new Font(config.MonospaceFontPath);
+                    asset = Helpers.Helpers.TMPFontFromUnityFont(uFont);
```

Comment:
Is there a reason Helpers needed to be in its own namespace of the same name? Generally I'd do `Utilities.Helpers` to avoid this redundancy, or just have it be in the root namespace if its only the one class.